app/bundles/CampaignBundle/Config/config.php

```diff
@@ -27,6 +27,14 @@
                 'path'       => '/campaign/preview/{objectId}',
                 'controller' => 'Mautic\EmailBundle\Controller\PublicController::previewAction',
             ],
+            'mautic_campaign_countries_stats'      => [
+                'path'       => '/campaign-countries-stats/preview/{objectId}',
```

Comment:
I suggest to use the `/campaign/...` path for the URL. Maybe something like `/campaign/countries-stats/{objectId}`